packages/core/src/modules/credentials/CredentialsModule.ts

```diff
@@ -242,7 +246,13 @@ export class CredentialsModule implements CredentialsModule {
         comment: options.comment,
         autoAcceptCredential: options.autoAcceptCredential,
       }
-      const { message, credentialRecord } = await service.createRequest(record, requestOptions, connection.did)
+
+      const didDocument = await this.didResolver.resolveDidDocument(connection.did)
+
+      const key = await getPublicKeyBase58('Ed25519VerificationKey2018', didDocument)
+      const indyDid = indyDidFromPublicKeyBase58(key)
```

Comment:
There's other places where we create a request. Could you make sure to update all places. Also, would be good to add some tests that cover this.

Reply:
Used indy did in V1OfferCredentialHandler & V2OfferCredentialHandler
Test cases added